src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -538,3 +540,179 @@ export const sampleData: RawSpeciesStats = {
+    taxonomy_id: '511145',
+    strain: null,
+    database_version: '101.1',
+    common_name: 'Escherichia coli str. K-12 substr. MG1655',
+    scientific_name: null,
+    gencode_version: null,
+    assembly_name: 'ASM584v2',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'European Nucleotide Archive',
+      url: 'www.ensembl.org'
+    },
+    assembly_level: 'complete genome',
+    annotation_method: 'Generated from ENA annotation',
+    assembly_date: 'Sep 2013',
+    notes: []
+  },
+  caenorhabditis_elegans_GCA_000002985_3: {
+    id: 'GCA_000002985.3',
+    taxonomy_id: '6239',
+    strain: {
+      type: 'Strain',
+      value: 'N2'
+    },
+    database_version: '101.269',
+    common_name: 'Caenorhabditis elegans',
+    scientific_name: null,
+    gencode_version: null,
+    assembly_name: 'WBcel235',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'WormBase',
+      url: 'www.ensembl.org'
+    },
+    assembly_level: 'complete genome',
+    annotation_method: 'Import',
+    assembly_date: 'Feb 2013',
+    notes: []
+  },
+  plasmodium_falciparum_GCA_000002765_2: {
+    id: 'GCA_000002765.2',
+    taxonomy_id: '36329',
+    strain: null,
+    database_version: '101.1',
+    common_name: 'Plasmodium falciparum 3D7',
+    scientific_name: null,
+    gencode_version: null,
+    assembly_name: 'ASM276v2',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'Naval Medical Research Institute',
+      url: 'www.ensembl.org'
+    },
+    assembly_level: 'complete genome',
+    annotation_method: 'Import',
+    assembly_date: 'Apr 2016',
+    notes: []
+  },
+  saccharomyces_cerevisiae_GCA_000146045_2: {
+    id: 'GCA_000146045.2',
+    taxonomy_id: '4932',
+    strain: {
+      type: 'Strain',
+      value: 'S288C'
+    },
+    database_version: '101.4',
+    common_name: 'Brewers yeast',
+    scientific_name: 'Saccharomyces cerevisiae',
+    gencode_version: null,
+    assembly_name: 'R64',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'Saccharomyces Genome Database',
+      url: 'www.ensembl.org'
+    },
+    assembly_level: 'complete genome',
+    annotation_method: 'Import',
+    assembly_date: 'Dec 2014',
+    notes: []
+  },
+  triticum_aestivum_GCA_900519105_1: {
+    id: 'GCA_900519105.1',
+    taxonomy_id: '4565',
+    strain: {
+      type: 'Cultivar',
+      value: 'Chinese spring'
+    },
+    database_version: '101.4',
+    common_name: 'Wheat',
+    scientific_name: 'Triticum aestivum',
+    gencode_version: null,
+    assembly_name: 'IWGSC',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'International Wheat Genome Sequencing Consortium',
+      url: 'www.ensembl.org'
```

Comment:
https://www.ebi.ac.uk/ena/data/view/GCA_900519105.1

Reply:
That's done